lib/k8s-e2e-tests/tests/vector-agent.rs

```diff
@@ -830,6 +830,7 @@ async fn custom_selectors() -> Result<(), Box<dyn std::error::Error>> {
             kubernetes_logs:
               type: kubernetes_logs
               extra_label_selector: "my_custom_negative_label_selector!=my_val"
+              extra_namespace_label_selector: "my_custom_negative_label_selector!=my_val"
```

Comment:
I don't think we actually test the functionality of the selector in this test, we should either move this into it's own test (more clear on what actually failed) or update this test to ensure we validate the namespace selector in this test.

Reply:
Forgive my poor rust-foo. I admit, I was mostly doing copy-pasta.